packages/lms-client/src/repository/ModelSearchResultDownloadOption.ts

```diff
@@ -10,9 +10,27 @@ import { type ModelSearchResultDownloadOptionData } from "@lmstudio/lms-shared-t
 import { type ModelSearchResultDownloadOptionFitEstimation } from "@lmstudio/lms-shared-types/dist/repository/ModelSearch";
 import { z } from "zod";
 
+/**
+ * @public
+ */
+export interface DownloadOnProgressCallbackParam {
```

Comment:
Nit: maybe...
```suggestion
export interface DownloadProgressUpdate {
```

?